kore/app/format/Main.hs

```diff
@@ -52,7 +52,12 @@ infoMod =
 main :: IO ()
 main =
     do
-        options <- mainGlobal (ExeName "kore-format") commandLine infoMod
+        options <-
+            mainGlobal
+                (ExeName "kore-format")
+                Nothing
```

Comment:
Maybe there should be a comment here describing what the `Nothing` is like in `parser/Main.hs`?